packages/dev-server-hmr/src/HmrPlugin.ts

```diff
@@ -0,0 +1,184 @@
+import { Plugin, WebSocketsManager } from '@web/dev-server-core';
+import type { ServerArgs } from '@web/dev-server-core/dist/Plugin';
```

Comment:
This import won't work now that we're using package exports map, node will throw. I'll add an export for this type.